code/src/layouts/modals/auth/AuthModal.tsx

```diff
@@ -1,14 +1,48 @@
+import { useState } from "react";
+
 import CloseIcon from "@mui/icons-material/Close";
+
 import AppBox from "@/components/app-box/AppBox";
 import AppIconButton from "@/components/app-icon-button/AppIconButton";
 import AppTypography from "@/components/app-typography/AppTypography";
+import AppButton from "@/components/app-button/AppButton";
+
+import LoginForm from "@/layouts/modals/auth/components/login-form/LoginForm";
+import SignupForm from "@/layouts/modals/auth/components/signup-form/SignupForm";
+
 import { useModalContext } from "@/context/ModalContext";
 
 import "@/layouts/modals/auth/AuthModal.scss";
 
 const AuthModal = () => {
   const { closeModal } = useModalContext();
 
+  const [isLoginPage, setIsLoginPage] = useState(true);
```

Comment:
Great work! Looks nice!
Also, I suggest using a string (use typescript for it to make the possibility of passing just our exact tabs )for this state and calling it an activeTab. Because in future maybe we will have some another tab